src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -95,5 +95,5 @@
       "Size": 1904
     }
   },
-  "PackageSize": 2632010
+  "PackageSize": 2685258
```

Comment:
*Probably* not related to this PR, but why'd the package size increase so much?

Reply:
Looks like its 

```
"assemblies/System.Private.CoreLib.dll": {
      "Size": 516811
      "Size": 536436
    },
```

or at least a fair chunk of it will be